projects/templatetags/projects_extras.py

```diff
@@ -0,0 +1,80 @@
+from django import template
+from projects.models import *
+
+register = template.Library()
+
+
+@register.filter
+def get_bundles(project):
+    return BundleDetail.objects.filter(project__id=project.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_builds(bundle):
+    return BuildDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')
+
+
+@register.filter
+def get_suite(bundle):
+    return SuiteDetail.objects.filter(bundle__id=bundle.id).exclude(status='ARCHIVED')[0]
```

Comment:
would there ever be more than 1 suite in a bundle? If so that might can an issue